.github/workflows/ci.yml

```diff
@@ -295,6 +295,14 @@ jobs:
       - name: cargo test compile
         run: cargo test --workspace ${{ env.NO_WASM_PKGS }} --locked --target wasm32-unknown-unknown --all-features --no-run
 
+      - name: install wasm-pack
+        run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
```

Comment:
We can use the install-action we already use for this (see https://github.com/taiki-e/install-action/blob/main/TOOLS.md)